src/Rings/MPolyQuo.jl

```diff
@@ -36,11 +36,22 @@ function groebner_assure(r::MPolyQuoRing)
   return true
 end
 
-function show(io::IO, Q::MPolyQuoRing)
-  Hecke.@show_name(io, Q)
-  Hecke.@show_special(io, Q)
-  io = IOContext(io, :compact => true)
-  print(io, "Quotient of $(base_ring(Q)) by $(modulus(Q))")
+function Base.show(io::IO, ::MIME"text/plain", Q::MPolyQuoRing)
+   println(io, "Quotient")
+   println(io, "  of ", base_ring(Q))
+   print(io, "  by Ideal with " , ngens(modulus(Q)), "generators")
```

Comment:
The key information of the quotient ring is the base_ring and the modulus. 

I agree that in standard view only the number of generators is printed, but there should be an even more detailed view available, which prints the modulus (and not in supercompact printing of an ideal). Think of a user having defined several quotients of the same base_ring and trying to pick the right one for the next computation.

Reply:
Agreed.

Reply:
Indeed, this also solves the dilemma of how much information is necessary.